flight_booker/src/Main.gren

```diff
@@ -114,48 +120,66 @@ view model =
             , isDisabled = model.flightType == OneWay
             }
         , bookButton
-            { isDisabled = checkIfDisabled model.flightType model.date1 model.date2 }
+            { isDisabled = checkIfDisabled model.flightType model.date1 model.date2
+            }
         ]
 
 
 flightTypeSelector : Html Msg
 flightTypeSelector =
-     select
+    select
         [ id "flight-type"
         , onInput FlightTypeChanged
         ]
-        [ option [] [ text ( flightTypeToString OneWay ) ]
-        , option [] [ text ( flightTypeToString ReturnFlight ) ]
+        [ option
+            []
+            [ text (flightTypeToString OneWay)
+            ]
+        , option
+            []
+            [ text (flightTypeToString ReturnFlight)
+            ]
         ]
 
 
 type alias DateInputConfig =
-    { id: String
-    , value: String
-    , date: Maybe Date
-    , changeMsg: String -> Msg
-    , isDisabled: Bool
+    { id : String
+    , value : String
+    , date : Maybe Date
+    , changeMsg : String -> Msg
+    , isDisabled : Bool
     }
 
-dateInput: DateInputConfig -> Html Msg
+
+dateInput : DateInputConfig -> Html Msg
 dateInput { id = id_, value = value_, date, changeMsg, isDisabled } =
     input
         [ id id_
         , value value_
         , onInput changeMsg
         , disabled isDisabled
-        , style "background-color" ( if date == Nothing then "red" else "white" )
+        , style
+            "background-color"
+            (if date == Nothing then
+                "red"
+             else
+                "white"
+            )
         ]
         []
 
 
-bookButton : { isDisabled : Bool } -> Html Msg
+bookButton :
+    { isDisabled : Bool
+    }
+    -> Html Msg
```

Comment:
IMHO, type signatures containing records with at most one property, looks nicer when formatted on a single line.

Reply:
:+1: Yeah, I was thinking it would be a good rule for arrays and records (for types, and expressions, and maybe also patterns?) that if there's only a single element, prefer single-line formatting.